subgraph-triggers/ethereum-explorer-transactions/README.md

```diff
@@ -0,0 +1,19 @@
+## Subtreams Triggers - Importing transactions from the Ethereum Explorer package.
+
+This a basic example of importing a Substreams module into a subgraph using Substreams triggers. The `map_filter_transactions` module of the [Ethereum Explorer](https://github.com/streamingfast/substreams-explorers/tree/main/ethereum-explorer) package is imported and a `Transaction` GraphQL entity is created for every transaction received.
+
+## Getting Started
+
+1. Install the dependencies:
+
+```bash
+npm install
+```
+
+2. Generate the Protobuf of the Ethereum Explorer Substreams:
+
+```bash
+buf generate --exclude-path="google" ethereum-explorer-v0.1.2.spkg#format=bin
```

Comment:
`buf generate --path my/transaction/transaction.proto` ?

perhaps exclude `--exclude-path="sf/substreams"` .. so we generate less